bench/main.ml

```diff
@@ -90,8 +101,13 @@ let input =
   in
   Arg.(value & opt options `All & info [ "b"; "bench" ] ~doc)
 
+let json_output =
```

Comment:
Should probably this in a `cli.ml` file to avoid duplication from `db_bench`. We can do a later refactoring pass to ensure that the two Cmdliner APIs for `main` and `db_bench` are fully consistent.